python/rapidsmpf/rapidsmpf/examples/dask.py

```diff
@@ -106,8 +128,9 @@ def extract_partition(
 
 def dask_cudf_shuffle(
     df: dask_cudf.DataFrame,
-    shuffle_on: list[str],
+    on: list[str],
```

Comment:
https://github.com/rapidsai/rapidsmpf/blob/branch-25.06/docs/source/quickstart.md?plain=1#L29 will need to be updated from `shuffle_on` to `on` I think.